config/samples/custom-theme/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+resources:
+- ../base/openstack
+
+patches:
+- path: custom_theme.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+configMapGenerator:
+- files:
+  - ./_11_rcue_theme.py
+  - ./rcue.tar.gz
```

Comment:
I might need an additional iteration to add `rcue.tar.gz` sample that can be used by the users.